codec-http/src/main/java/io/netty/handler/codec/http/multipart/HttpPostMultipartRequestDecoder.java

```diff
@@ -805,6 +795,35 @@ private InterfaceHttpData findMultipartDisposition() {
         }
     }
 
+    private Attribute getContentDispositionAttribute(String... values) {
+        Attribute attribute;
```

Comment:
Why is it necessary to declare the `attribute` here?

Reply:
That is how it was before my patch on [4.1](https://github.com/dminkovsky/netty/blob/4.1/codec-http/src/main/java/io/netty/handler/codec/http/multipart/HttpPostMultipartRequestDecoder.java#L708). For the actual test/patch, please see my first commit (https://github.com/netty/netty/pull/7265/commits/e4a98530078894b1132a929fe007c7e7b854a9d7). My second commit, which this comes from, was done because Netkins complained I had gone above 5 indents and should refactor. I would obviously rather not include this commit in this PR.

Reply:
Ok. But now there is no need for this. Just use fast return without variable:
```java
    return factory.createAttribute(request, name, value);
```

Reply:
Yes of course. Fixed this.